bundles/org.openhab.binding.boschshc/src/test/java/org/openhab/binding/boschshc/internal/devices/AbstractBatteryPoweredDeviceHandlerTest.java

```diff
@@ -31,7 +30,6 @@
  *
  * @param <T> type of the battery-powered device to be tested
  */
-@NonNullByDefault
```

Comment:
I think it was removed due to the following compilation error:

```
Null constraint mismatch: The type 'T extends org.openhab.binding.boschshc.internal.devices.AbstractSmokeDetectorHandler' is not a valid substitute for the type parameter 'T extends org.openhab.binding.boschshc.internal.devices.@NonNull AbstractBatteryPoweredDeviceHandler'
```

But I think I found a way to fix this (will push a commit soon).

Reply:
I checked, and I'm guessing it was related to this error: 
```
AbstractSmokeDetectorHandlerTest.java:[45,57] Null constraint mismatch: The type 'T extends org.openhab.binding.boschshc.internal.devices.AbstractSmokeDetectorHandler' is not a valid substitute for the type parameter 'T extends org.openhab.binding.boschshc.int`ernal.devices.@NonNull AbstractBatteryPoweredDeviceHandler
```

And then this when applying annotation in AbstractSmokeDetectorHandlerTest also: 
```
The @NonNull field smokeDetectorCheckStateCaptor may not have been initialized
```

I will provide a fix for that since I know you are currently away.

Reply:
ok, you were faster 😉